addons/website/views/res_config_views.xml

```diff
@@ -7,134 +7,384 @@
             <field name="model">website.config.settings</field>
             <field name="arch" type="xml">
                 <form string="Configure Website" class="oe_form_configuration">
-                    <header>
-                        <button string="Apply" type="object" name="execute" class="oe_highlight"/>
+                    <header class="o_header_sticky">
+                        <button string="Save" type="object" name="execute" class="oe_highlight"/>
                         <button string="Cancel" type="object" name="cancel" class="oe_link"/>
                     </header>
-                    <div>
-                        <field name="website_id" invisible="1"/>
-                        <group name="google_analytics" string="Domain">
-                            <field name="website_name"/>
-                            <label for="google_analytics_key"/>
-                            <div name="google_analytics_key">
-                                <div>
-                                    <div name="google_analytics_key" class="oe_inline">
-                                        <field name="google_analytics_key" placeholder="UA-XXXXXXXX-Y"/>
+                    <sheet>
+                        <div>
+                            <field name="website_id" invisible="1"/>
+                            <h2 class="mt32">Webmaster</h2>
```

Comment:
Not sure, but wouldn't it be better to have classes (group_header, box_container, ...) instead of directly putting css-based classes ? I am not a good css guy, so maybe ask @qsm-odoo or smart people like him to be sure.